docs/docsite/.templates/banner.html

```diff
@@ -40,7 +41,7 @@
         /* temp extra banner to advertise something */
         banner += extra_banner;
 
-        msg += 'This is the <b>latest</b> (stable) community version of the Ansible documentation. For Red Hat customers, see <a href="https://www.ansible.com/compare"> the difference between Ansible community projects and Red Hat supported products</a> or <a href="https://access.redhat.com/support/policy/updates/ansible-automation-platform">Ansible Automation Platform Life Cycle</a> for subscriptions.';
+        msg += 'This is the <b>latest</b> (stable) community documentation. Red Hat customers, see <a href="https://access.redhat.com/support/policy/updates/ansible-automation-platform">Ansible Automation Platform Life Cycle</a>';
```

Comment:
The wording here feels kind of weird. It's like the "Red Hat customers" bit is a random fragment or something and we're not addressing the purpose. I also don't like the word "customers" here but that might just be my personal preference.

I'm thinking something like:

For Red Hat Ansible Automation Platform subscriptions, refer to the "Life Cycle" page for version details.

Reply:
The goal here is to keep the sentence to one line so the banner takes up less space. I'll experiment a bit with the wording and see if I can add some of your suggestions.